components/nimbus/src/nimbus.udl

```diff
@@ -74,7 +74,6 @@ enum NimbusError {
     "DatabaseNotReady",
 };
 
-[Threadsafe]
```

Comment:
In UniFFI v0.11.0 the `[Threadsafe]` attribute is now the default and only way to safely expose objects, and as such is deprecated.